Lil-Peep/src/styles/modules/featured-categories.scss

```diff
@@ -1,19 +1,18 @@
 @media only #{ $mobile-view }{
   .categories-main {
+    background-color: $color-bg;
+    border-bottom: 1px solid $color-bd;
```

Comment:
Properties should be ordered background-color, border, border-bottom, border-color, border-style, font-family